mito-ai/src/icons/DeleteIcon.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const DeleteIcon : React.FC = () => {
+    return (
+        <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24">
+            <path d="M6 19c0 1.1.9 2 2 2h8c1.1 0 2-.9 2-2V7H6zM19 4h-3.5l-1-1h-5l-1 1H5v2h14z" fill="black" className="jp-icon3 jp-icon-selectable"/>
```

Comment:
I think we should update this to `<g fill="currentColor">` instead of hardcoding to black? Although, it does already seem responsible to light and dark mode is that because of `className="jp-icon3 jp-icon-selectable"`?